packages/react-core/src/components/Nav/examples/Nav.md

```diff
@@ -457,3 +457,1155 @@ class NavTertiaryList extends React.Component {
   }
 }
 ```
+
+### Default nav
```

Comment:
Super close! Just move all these demos to a new `react-core/src/demos/Nav.md` file.